submarine-serve/src/test/java/org/apache/submarine/serve/tensorflow/SeldonTFServingTest.java

```diff
@@ -0,0 +1,70 @@
+// /*
+//  * Licensed to the Apache Software Foundation (ASF) under one
+//  * or more contributor license agreements.  See the NOTICE file
+//  * distributed with this work for additional information
+//  * regarding copyright ownership.  The ASF licenses this file
+//  * to you under the Apache License, Version 2.0 (the
+//  * "License"); you may not use this file except in compliance
+//  * with the License.  You may obtain a copy of the License at
+//  *
+//  *   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why do you comment all the lines in the file?

Reply:
Sorry, I accidentally pushed this file.